packages/wdio-allure-reporter/tests/cucumber.suite.test.ts

```diff
@@ -470,23 +486,29 @@ describe('reporter option "useCucumberStepReporter" set to true', () => {
             reporter.onHookStart(cucumberHelper.hookStart())
             reporter.addAttachment(attachmentHelper.xmlAttachment())
             reporter.onHookEnd(cucumberHelper.hookEnd())
+
             const suiteResults: any = { tests: [cucumberHelper.testPass()], hooks: new Array(2).fill(cucumberHelper.hookEnd()) }
+
             reporter.onSuiteEnd(cucumberHelper.scenarioEnd(suiteResults))
             reporter.onSuiteEnd(cucumberHelper.featureEnd(suiteResults))
             reporter.onRunnerEnd(runnerEnd())
 
-            const results = getResults(outputDir)
-            expect(results).toHaveLength(2) // one for report, one for attachment
-            allureXml = results.find((xml: any) => xml('ns2\\:test-suite').length >= 1)
+            const { results } = getResults(outputDir)
+
+            allureResult = results[0]
+
+            expect(results).toHaveLength(1)
```

Comment:
please assert length before you get value

Reply:
:+1: